packages/ccdi-cde/src/v1/sample/library_source_material.rs

```diff
@@ -0,0 +1,183 @@
+use introspect::Introspect;
+use rand::distributions::Standard;
+use rand::prelude::Distribution;
+use serde::Deserialize;
+use serde::Serialize;
+use utoipa::ToSchema;
+
+use crate::CDE;
+
+/// **`caDSR CDE 14808227 v1.00`**
+///
+/// This metadata element is defined by the caDSR as "The cellular source of
+/// the double stranded DNA fragments analyzed by high-throughput sequencing.".
+///
+/// Link:
+/// <https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14808227%20and%20ver_nr=1>
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize, ToSchema, Introspect)]
+#[schema(as = cde::v1::sample::LibrarySourceMaterial)]
+pub enum LibrarySourceMaterial {
+    /// `Bulk Cells`
+    ///
+    /// * **VM Long Name**: Bulk Cell Specimen
+    /// * **VM Public ID**: 7592130
+    /// * **Concept Code**: C178223
+    /// * **Begin Date**:   03/12/2024
+    ///
+    /// A biospecimen consisting of multiple cells intended to be analyzed as a pool.
+    #[serde(rename = "Bulk Cells")]
+    BulkCells,
+
+    /// `Bulk Nuclei`
+    ///
+    /// * **VM Long Name**: Bulk Nucleus Specimen
+    /// * **VM Public ID**: 7592129
+    /// * **Concept Code**: C178224
+    /// * **Begin Date**:   02/28/2024
+    ///
+    /// A biospecimen consisting of multiple nuclei intended to be analyzed as a pool.
+    #[serde(rename = "Bulk Nuclei")]
+    BulkNuclei,
+
+    /// `Bulk Tissue`
+    ///
+    /// * **VM Long Name**: Bulk Tissue Specimen
+    /// * **VM Public ID**: 7592128
+    /// * **Concept Code**: C178225
+    /// * **Begin Date**:   02/28/2024
+    ///
+    /// A biospecimen either derived from a whole tissue specimen or tissue section, which may consist of heterogeneous cells or tissues.
+    #[serde(rename = "Bulk Tissue")]
+    BulkTissue,
+
+    /// `Single-cells`
+    ///
+    /// * **VM Long Name**: Single Cell Suspension
+    /// * **VM Public ID**: 14838800
+    /// * **Concept Code**: C204464
+    /// * **Begin Date**:   03/12/2024
+    ///
+    /// A dilute suspension of cells intended to be further fractionated for assays focused on single-cells.
+    #[serde(rename = "Single-cells")]
+    SingleCells,
+
+    /// `Single-nuclei`
+    ///
+    /// * **VM Long Name**: Single Nucleus Suspension
+    /// * **VM Public ID**: 14838802
+    /// * **Concept Code**: C204465
+    /// * **Begin Date**:   03/12/2024
+    ///
+    /// A dilute suspension comprised of isolated intact cell nuclei intended to be further fractionated for assays focused on single-nuclei.
+    #[serde(rename = "Single-nuclei")]
+    SingleNuclei,
+
+    /// `Not Reported`
+    ///
+    /// * **VM Long Name**: Not Reported
+    /// * **VM Public ID**: 5612322
+    /// * **Concept Code**: C43234
+    /// * **Begin Date**:   03/01/2024
+    ///
+    /// Not provided or available.
+    #[serde(rename = "Not Reported")]
+    NotReported,
+
+    /// `Other`
```

Comment:
I don't see an "Other" in the [permissible values listed on on the caDSR site ](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14808227%20and%20ver_nr=1) - is this something the harmonization team agreed to add and I missed it in the discussions? If not, let's remove it (here and in the subsequent lines in this file).

Reply:
I noticed that. I picked the list from the Github conversation [here](https://github.com/CBIIT/ccdi-federation-api/discussions/114#discussioncomment-10938259)

Reply:
I agree that "Other" is not a PV for this CDE.  I asked @dunnpa to comment on whether this PV is going to be added.  If not, we should not include "Other" in our implementation.

Reply:
@grugna 
Per [discussion](https://github.com/CBIIT/ccdi-federation-api/discussions/114#discussioncomment-11041032), there is not a plan to add "Other" as a permissible value to this CDE.

Maureen also commented in an email - "In general, our practice is not to add ‘Other’ as a PV.  We would rather additional PVs as the use cases arise."

Thank you @e-t-k @grugna for your work on this.